src/styles/card.css

```diff
@@ -0,0 +1,147 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: flex-start;
+  width: fit-content;
+  padding: 16px;
+
+  background: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  --color-navy-blue: #060b35;
+  --color-blue: #00acdc;
+  --color-light-gray: #616070;
+}
+
+.card__image {
+  display: block;
+  height: 134px;
+  width: 160px;
+  margin: 16px auto 0;
+}
+
+.card__text {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  margin: 0;
+}
```

Comment:
why applying `flex` to `<p` tags?
for the container of two `<p` - yes, why not. But for each individual `<p` - disagree.

<img width="689" alt="p flex" src="https://user-images.githubusercontent.com/39308717/200528231-f87fe4b8-0e5d-4d03-b59c-478888c06fe5.png">